wix/Build.OpenJDK_generic.cmd

```diff
@@ -80,6 +82,10 @@ SET WIN_SDK_FULL_VERSION=10.0.17763.0
 SET WORKDIR=Workdir\
 mkdir %WORKDIR%
 
+@REM Add necessary wix extensions here
```

Comment:
I think my preference here would be to install these at an infra level rather than everytime. Perhaps add this to the GitHub workflow and then I'll make sure that the extensions exist on our CI machines along with the updated wix V5?

Reply:
I can try this, although I have 2 thoughts:
1) Doing this at the infra level means that all vendors would need to read the README and add this themselves or the code will break
1) This command adds the extensions to a local, project specific subdirectory. I would believe this would mean that most build scenarios would have to install this every time anyways

Reply:
ahh I hadn't realised that was the case. In which case maybe it's better staying where it is